application/src/main/java/be/jpendel/application/SignupUC.java

```diff
@@ -11,6 +11,8 @@ public SignupUC(MemberRepository memberRepository) {
         this.memberRepository = memberRepository;
     }
 
+    //TODO brrrr 'new Member' ipv een builder
+    // Volgens mijn begrip mag de applicatie laag niet een Member rechtreeks aanmaken ..waar is CreateMemberCommand ?
```

Comment:
Dit zou idd een builder of een factory moeten zijn, maar buiten de creates komen die Members eigenlijk altijd uit de repository.